File: client/src/app/_models/chapterDto.ts
import { ApprovalStatusChapter } from "../_extensions/enumHelper";

export interface ChapterDto {
  id: number | null;
  name: string;
  creationTime: Date;
  updateTime: Date | null;
  view: number;
  status: boolean;
  approvalStatus: ApprovalStatusChapter;
}

File: API/Dtos/ChapterDto.cs
using API.Helpers;

namespace API.Dtos
{
    public class ChapterDto
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public bool Status { get; set; }
        public DateTime CreationTime { get; set; }
        public DateTime? UpdateTime { get; set; }
        public int View { get; set; }
        public ApprovalStatusChapter ApprovalStatus { get; set; }
        public int Rank { get; set; }
    }
}